jsonschema/workflow_execution.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin/workflow-execution-event-request",
+  "properties": {
+    "RequestId": {
+      "type": "string"
+    },
+    "Event": {
+      "type": "object",
+      "properties": {
+        "execution_id": {
+          "$ref": "#/$defs/WorkflowExecutionIdentifier"
+        },
+        "producer_id": {
+          "type": "string"
+        },
+        "phase": {
+          "type": "integer"
+        },
+        "occurred_at": {
+          "$ref": "#/$defs/Timestamp"
+        },
+        "output_result": {
+          "oneOf": [
```

Comment:
is oneOf a jsonschema thing also?  or just a proto thing?

Reply:
yeah, jsonschema also has **oneOf**. https://json-schema.org/understanding-json-schema/reference/combining.html#oneof